themes/digital.gov/layouts/events/card-event-past.html

```diff
@@ -1,49 +1,52 @@
 {{ $path := "" }}{{ with .File }}{{ $path = .Path }}{{ else }}{{ $path = .Path }}{{ end }}
-<div class="card card-event card-event-past card-linked" data-edit-this="{{- $path -}}" {{ if .Params.short_url }}data-short_url="{{- .Params.short_url -}}"{{ end }}>
-  <div class="grid-row tablet:grid-gap-4">
 
-    {{- if .Params.youtube_id -}}
-    <div class="grid-col-12 tablet:grid-col-4 tablet:order-last">
-      <div aria-role="img" class="media-featured youtube-player" data-id="{{ .Params.youtube_id }}" data-link="{{ .Permalink }}"></div>
-    </div>
-    {{- else -}}
-    <div class="non-youtube-event-card grid-col-12 tablet:grid-col-4 tablet:order-last grid-row">
-      <div class="grid-row flex-align-self-center" data-link="{{ .Permalink }}">
-        <svg xmlns="http://www.w3.org/2000/svg" role="img" alt="" aria-hidden="true" height="150" width="150" viewBox="0 0 25 25">
-          <title>Calendar Event</title>
-          <path d="M0 0h24v24H0z" fill="none"/>
-          <path fill="#005ea2" d="M17 12h-5v5h5v-5zM16 1v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2h-1V1h-2zm3 18H5V8h14v11z"/>
-        </svg>
-      </div>
-    </div>
-    {{- end -}}
-
-    <div class="grid-col-12 tablet:grid-col-8">
+{{/*  Unique ID to link article to its title in screen readers  */}}
+{{ $titleID := (printf "%s%s" "title-" .Title) | urlize }}
```

Comment:
Hugo filter [`urlize`]( https://gohugo.io/functions/urlize/) to convert title into a valid ID